### ISSUE TITLE
has unit appears as a slot for Biosample

### ISSUE THREAD

**Issue by turbomam** [microbiomedata/nmdc-schema#1723]
look at all of these slots with spaces in their names in Biosample!

```
      - has numeric value
      - has raw value
      - has unit
```

the search in the online documentation doesn't make it easy to see that there are `has_unit` and `has unit` slots or which classes use them.

**Comment by turbomam** [comment]
`poetry run linkml-lint --format tsv --output nmdc.yaml.lint.tsv src/schema/nmdc.yaml` does not catch those slots with whitespace in their names.

PS we could have a TSV liniting target in project.Makefile in addition to the basic one in Makefile

**Comment by turbomam** [comment]
see also 
- #1724

**Comment by turbomam** [comment]
@sujaypatil96 observed that it also appears in mixs.yaml

I **think** that once the slot assignment is deleted from `Biosample`, its presence in `mixs.yaml` won't re-inject the assignment back into Biosample.

But removing it form `mixs.yaml` **is** preferable. Having said that, we will be switching to importing from GSC's `mixs.yaml` (v6.2.0) soon, replacing our current practice of importing from our static mixs.yaml.

**Comment by turbomam** [comment]
- removed the association between the `has ...` slots and `Biosample` in `nmdc.yaml`
- added deletion commands to `assets/yq-for-mixs_subset_modified.txt` so that the `has ...` slots  won't get into `src/schema/mixs.yaml` in the first place
